www/src/components/sidebar/section-title.js

```diff
@@ -166,11 +164,7 @@ const SplitButton = withI18n()(({ i18n, itemRef, item, uid }) => {
       <button
         aria-controls={uid}
         aria-expanded={isExpanded}
-        aria-label={
-          isExpanded
-            ? i18n._(t`${item.title} collapse`)
-            : i18n._(t`${item.title} expand`)
-        }
+        aria-label={i18n._(item.title)}
```

Comment:
After talking with @madalynrose we think the best title for this would be `${item.title} toggle` to highlight the fact that it's a toggle.

Reply:
Great, pushing now.